web/static/frontend/css/machines-layout.scss

```diff
@@ -8,21 +8,16 @@
                    'copy'
                    'regen'
                    'edit'
+                   'deact'
                    'delete'
                    / 1fr;
     grid-gap: 20px;
 
     @media #{$sm} {
-      grid-template: 'name name  name  name  '
-                     'key  key   key   key   '
-                     'copy regen edit  delete'
-                    / auto auto  auto  auto;
-    }
-
-    @media #{$lg} {
-      grid-template: 'name   name   name   name   name  '
-                     'key    copy   regen  edit   delete'
-                    / 1fr    auto   auto   auto   auto;
+      grid-template: 'name   name   name   name  '
```

Comment:
fr units not allowed on `grid-template`; must be one of (%, Hz, ch, cm, deg, dpcm, dpi, dppx, em, ex, grad, in, kHz, mm, ms, pc, pt, px, q, rad, rem, s, turn, vh, vmax, vmin, vw)